google/cloud/forseti/services/dao.py

```diff
@@ -1004,13 +1004,17 @@ def add_role_by_name(cls, session, role_name, permission_names):
         @classmethod
         def delete_role_by_name(cls, session, role_name):
             """Deletes a role by name."""
-
             LOGGER.info("Deleting an existing role, role_name = %s,"
                         " session = %s", role_name, session)
-            session.query(Role).filter(Role.name == role_name).delete()
-            role_permission_delete = role_permissions.delete(
-                role_permissions.c.roles_name == role_name)
-            session.execute(role_permission_delete)
+            bindings_tbd = [binding for binding in
```

Comment:
What is the abbreviation `tbd`?  Is it `to_be_deleted`?  If yes, would you mind to rename this var to `bindings_to_be_deleted` so that is much easier for others to read the code?

Reply:
sure, let me change that